File: updraft-sdk/src/main/java/com/appswithlove/updraft/api/request/FeedbackEnabledRequest.java
package com.appswithlove.updraft.api.request;

import com.google.gson.annotations.SerializedName;

public class FeedbackEnabledRequest {

    @SerializedName("sdk_key")
    private String mSdkKey;

    @SerializedName("app_key")

    private String mAppKey;

    public String getSdkKey() {
        return mSdkKey;
    }

    public void setSdkKey(String sdkKey) {
        mSdkKey = sdkKey;
    }

    public String getAppKey() {
        return mAppKey;
    }

    public void setAppKey(String appKey) {
        mAppKey = appKey;
    }

}

File: updraft-core/src/commonMain/kotlin/com/appswithlove/updraft/api/request/FeedbackEnabledRequest.kt
package com.appswithlove.updraft.api.request

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class FeedbackEnabledRequest(
    @SerialName("sdk_key") val sdkKey: String? = null,
    @SerialName("app_key") val appKey: String? = null,
)
